README.md

```diff
@@ -0,0 +1,55 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+Description the project.
+
+> Testing the linters
+
+## Built With
+
+- html
+- css
+- git
+- webhint
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Clone repo
+git clone
```

Comment:
- [ ] Please make sure you write complete information in your readme.  Please write the link used to clone the project 

e.g `git clone https://github.com/belalgamal46/Hello-Microverse.git`